EFCore.SqlServer.HierarchyId/EFCore.SqlServer.HierarchyId.csproj

```diff
@@ -51,5 +52,29 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  
+  <Target Name="SetNuspecProperties" BeforeTargets="GenerateNuspec">
+    <PropertyGroup>
+      <EFCoreSqlServerVersion>@(PackageReference->WithMetadataValue('Identity', 'Microsoft.EntityFrameworkCore.SqlServer')->Metadata('Version'))</EFCoreSqlServerVersion>
```

Comment:
Bet you didn't know that was possible in MSBuild 😉

Reply:
Whoa... wizardry...